### PR TITLE
Added fix so rheostat won't break in React v15

### PR DESCRIPTION
**Pull request by marsbergen**

### CODE REVIEW

File: src/Slider.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import linear from './algorithms/linear';
 
 // istanbul ignore next
 function getDOMNode(node) {
-  return React.version.indexOf('0.14') > -1 ? node : node.getDOMNode();
+  return _react2['default'].version.replace('.', '') >= 140 ? node : node.getDOMNode();
```
Comment (ljharb) [inline src/Slider.jsx]: `_react2['default']` looks like babel output. I think this needs to stay `React.version`.

Comment (marsbergen) [inline src/Slider.jsx]: Ah shit! Sorry, it's a bit too early to do PR :) I'll fix it right away!

File: src/Slider.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import linear from './algorithms/linear';
 
 // istanbul ignore next
 function getDOMNode(node) {
-  return React.version.indexOf('0.14') > -1 ? node : node.getDOMNode();
+  return React.version.replace('.', '') >= 140 ? node : node.getDOMNode();
```
Comment (ljharb) [inline src/Slider.jsx]: I'm wondering if this is actually the best way to check if it's 0.14 or 15.0 or newer :-/

**Comment by goatslacker** [comment]
We should probably drop support for 0.13.x

**Comment by ljharb** [comment]
I don't think we should drop support for 0.13 at least until Facebook has released 16. In other words, I think we should always support "latest + 2 versions back".

**Comment by marsbergen** [comment]
Shall I wait until there is a final call, if you want to drop or keep support for 0.13.x?

**Comment by goatslacker** [comment]
Why 2 versions? +1 seems good enough especially since deprecations usually happen between version. 

https://facebook.github.io/react/blog/2016/02/19/new-versioning-scheme.html#avoiding-the-major-cliff

**Comment by ljharb** [comment]
We can discuss it offline, but that's the policy enzyme is maintaining already, and it's also the policy that node itself maintains. 2 versions because providing an upgrade path is critical, and just because 16 comes out doesn't mean everybody will have stopped using 0.14.

**Comment by goatslacker** [comment]
Yeah but node follows a [different version scheme](https://en.wikipedia.org/wiki/Linux_kernel#Version_numbering).

You could also say the same case about people and React 0.13 when v16 comes out. I think most people are on v15 whenever 16 comes out and if they're not then they can just use an older version of this package (we can even backport bug fixes if we need to).

Enzyme is also doing that weird package.json hack so that's probably why it's maintaining a -2 version path. I'm not necessarily tied to keeping that hack around throughout versions.

**Comment by ljharb** [comment]
That version scheme stopped applying once io.js/node 4 came out.

Enzyme is explicitly maintaining support for "latest minus 2", and will continue to do so. We'll drop 0.13 once v16 comes out. I strongly prefer rheostat to do the same.

**Comment by mikeodell77** [comment]
Would you describe what is broken? I am new to using this component (I am using React 15.1.0).

**Comment by mikeodell77** [comment]
Since my last comment, I see what is broken. Any ideas as to when the pull request will be merged in?

File: src/Slider.jsx (hunk 2 above)
Comment (goatslacker) [inline src/Slider.jsx]: This is not going to work for 0.14.x

Comment (goatslacker) [inline src/Slider.jsx]:
```js
const version = React.version.split('.');
return (version[1] === '14' || version[0] === '15') ? node : node.getDOMNode();
```

Comment (ljharb) [inline src/Slider.jsx]: you'd need `.replace(/\./g, '')` probably

**Comment by ljharb** [comment]
Closing this in favor of #26.

**Pull request closed without merging**